packages/manifests/direct/manifest.yaml

```diff
@@ -0,0 +1,342 @@
+---
```

Comment:
I guess this file could be regenerated by some simple script which cats appropriate manifests, right?

Reply:
Yes - happy to do that ... I'm guessing https://github.com/kubernetes/cloud-provider-gcp/tree/master/deploy is the canonical source?  I wanted to keep this relatively self-contained at first, and one gotcha for unification is the Pod vs Daemonset for the c-c-m, but I think we can relatively easily create a script/tool that could generate both forms if that's the direction we agree.

My desired outcome is that kOps consumes a manifest that cloud-provider-gcp tests with.  Today that isn't the case, and I'd love to get these aligned for kOps.   I think it matters because we have functionality like #393 that isn't tested by default - I think that's fine if it's a configuration nobody uses, but I'm hoping we can make a stronger statement in cloud-provider-gcp about what configurations people should use.

It might be that statement is "thou shalt run as a static pod", that's more work for me as a consumer in the short term, but clarity and using a cross-tested configuration is more important in the long run.